pkg/build/builder/sti.go

```diff
@@ -384,3 +393,13 @@ func scriptProxyConfig(build *api.Build) (*s2iapi.ProxyConfig, error) {
 	}
 	return config, nil
 }
+
+func copyArtifactSourceList(artifactsMapping []api.ImageSourcePath) (volumeList []s2iapi.VolumeSpec) {
```

Comment:
If you decided to use it, could I ask you to use empty `return` then and emphasize the fact of using named return value (like in the example that you provided)?